app/assets/javascripts/app-level.js

```diff
@@ -5,9 +5,9 @@
 $(document).ready(function() {
 	$(".moment-date-time").each(function(ind) {
 		$el = $(this);
-		var format = $el.data("format")  || "MMMM Do YYYY, h:mm a";
+		var format = $el.data("format")  || "YYYY-MM-DD HH:mm:ss";
```

Comment:
That's the default format for those that do not specify a format, which means timestamps like assessment submission time, job completion time, etc., and I thought it would be useful to have seconds in there to be precise (and I also changed it into a standard format that's easy to sort and manipulate I guess). For human friendly times like assessment due times, it won't display seconds.